packages/pipeline/src/scripts/update_radar_orderbook_usd_prices.ts

```diff
@@ -0,0 +1,83 @@
+import { logUtils } from '@0x/utils';
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+import * as ormConfig from '../ormconfig';
+import { handleError, isCli } from '../utils';
+
+if (isCli()) {
```

Comment:
I'm okay with this for now, but long-term we should implement proper logging with the ability to turn it on or off (and maybe even set different log levels).